coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/UDFTransformerTest.java

```diff
@@ -161,7 +161,7 @@ public void testNormalTransformation() {
         "{\"op\":\"+\",\"operands\":[{\"input\":0},{\"input\":3}]}", null, "targetFunc(icol * dcol, scol) + dcol");
 
     testFailedTransformation(tableOneQuery, tableOneConfig, targetUDF,
-        "[{\"op\":\"%\",\"operands\":[{\"input\":2},{\"input\":3}]}, {\"input\":1}]", null, null,
+        "[{\"op\":\"@\",\"operands\":[{\"input\":2},{\"input\":3}]}, {\"input\":1}]", null, null,
```

Comment:
Could you clarify that change that is being made here?

Reply:
Ok, it doesn't fit `testNormalTransformation`,  I moved it to another function `testUnsupportedOperatorTransformer`.  I think it might be more appropriate to put it in front of this function `testNoMatchOperatorTransformer`.
On the other hand, I just chose a symbol that I'm not likely to support. `@` is no other special meaning.